jax/_src/nn/functions.py

```diff
@@ -1008,9 +1032,20 @@ def _check_shape_and_dtype(t: Array | None, shape: Sequence[int],
         mask_type = MaskType.CAUSAL
       elif query_seq_lengths is not None:
         mask_type = MaskType.PADDING
+      # CuDNN supports only the left window with an exclusive boundary with
+      # causal mask is enabled.
```

Comment:
Nit: language doesn't make sense. Maybe 'with an exclusive boundary when causal mask is enabled'?

Reply:
Done.